src/mappergraphql/schema.graphql

```diff
@@ -1,26 +1,23 @@
 scalar Time # Equivalent of Go's time.Time provided by gqlgen
 
-input CaptureResultForSrcIp {
-    srcIp: String!
-    destinations: [Destination!]!
-}
-
 input Destination {
+    # Could be either IP addr or hostname
```

Comment:
We better separate our graph according to the actual type (IP/hostname) and keep a different destination type for each endpoint